docs/deployment/standalone-host.md

```diff
@@ -0,0 +1,62 @@
+# Installation to standalone host
+You can install Racetrack to a standalone host (eg. EC2 host or fresh VM instance)
+using the installer script that runs it on the Docker Engine infrastructure.
+
+## Requirements
+
+- Python 3.8+ with `pip` and `venv`
+- [Docker v20.10 (or higher)](https://docs.docker.com/engine/install/)
+  managed by a [non-root user](https://docs.docker.com/engine/install/linux-postinstall/#manage-docker-as-a-non-root-user)
+- [Docker Compose plugin](https://docs.docker.com/compose/install/linux/#install-using-the-repository)
+- curl
+
+For instance, on Debian-based systems do:
+```sh
+sudo apt update && sudo apt install curl python3 python3-pip python3-venv
+# Install user-managed docker
+curl -fsSL https://get.docker.com -o install-docker.sh
+sh install-docker.sh
+sudo usermod -aG docker $USER
+newgrp docker
+```
+
+## Install Racetrack
+Pick an installation directory:
+```sh
+mkdir -p ~/racetrack && cd ~/racetrack
+```
+and run
+```sh
+sh <(curl -fsSL https://raw.githubusercontent.com/TheRacetrack/racetrack/master/utils/standalone-wizard/runner.sh)
+```
+Follow the installation steps. Shortly after, your Racetrack instance will be ready.
+
+### Notes
+
+- Pay attention to the output. The installer will generate unique passwords for your setup,
+  you'll be provided with the Dashboard address and superuser credentials.
+- Set the environment variable `export RT_NON_INTERACTIVE=1`
```

Comment:
Is there a way to pass these questions as parameters so you can be non-interactive _and_ not go with the defaults? If yes, this should mention it, if no, we should maybe add that as an option.

Reply:
Good point. In fact there is only one parameter right now. Added such option and included it in docs